dotCMS/src/integration-test/java/com/dotcms/browser/BrowserAPITest.java

```diff
@@ -113,7 +117,98 @@ public static void prepare() throws Exception {
 
     }
 
-    
+    @Test()
+    public void test_GetFolderContent_pagination() throws DotDataException, DotSecurityException, IOException {
+
+        // create a folder
+        // create a 100 files
+        // paginate 10 in 100
+        final SiteDataGen   siteDataGen   = new SiteDataGen();
+        final FolderDataGen folderDataGen = new FolderDataGen();
+        final Host          host          = siteDataGen.nextPersisted();
+        final Folder        folder        = folderDataGen.site(host).nextPersisted();
+
+        for (int i = 0; i < 100; ++i) {
+
+            new FileAssetDataGen(FileUtil.createTemporalFile("test", ".txt", "this is a test")).host(host)
+                    .folder(folder).setPolicy(IndexPolicy.WAIT_FOR).nextPersisted();
+        }
+
+        Map<String, Object> resultMap = browserAPI.getFolderContent(BrowserQuery.builder()
+                .showDotAssets(true)
+                .showLinks(true)
+                .withHostOrFolderId(folder.getIdentifier())
+                .offset(0)
+                .showFiles(true)
+                .showFolders(true)
+                .showWorking(true)
+                .maxResults(10)
+                .build());
+
+        assertNotNull(resultMap);
+        assertEquals(100l, resultMap.get("total"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "total" appears 5 times in this file; the first occurrence is on line 149](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5476854)